### PR TITLE
413: remove validation messages from composite validator only if they…

### PR DESCRIPTION
**Pull request by sideisra**
… have the same validator code to avoid removal of validation message of other validators

### CODE REVIEW

**Comment by sideisra** [comment]
Solution for #413

File: mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/validation/CompositeValidationStatus.java [hunk 1]
```diff
@@ -73,25 +67,34 @@ void addMessage(Validator validator, List<? extends ValidationMessage> messages)
...
-		List<CompositeValidationMessageWrapper> messagesToRemove =
-				getMessagesInternal().stream()
-					.filter(messages::contains)  // only the given messages
-					.filter(message -> (message instanceof CompositeValidationMessageWrapper))
-					.map(message -> (CompositeValidationMessageWrapper) message)
-					.filter(message -> message.getValidatorCode().equals(System.identityHashCode(validator)))
-					.collect(Collectors.toList());
+	void removeMessage(final Validator validator, final List<? extends ValidationMessage> messages) {
+    final List<CompositeValidationMessageWrapper> messagesToRemove =
+        getMessagesInternal().stream()
```
Comment (manuel-mauky) [inline mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/validation/CompositeValidationStatus.java]: Can you please fix the indentation of this file?

**Comment by manuel-mauky** [comment]
Many thanks for the PullRequest. I've added a comment for a small fix to be done.

I will merge this PR into `release` branch instead of `develop`. This way this fix will be part of version `1.5.2` and you can test it with `1.5.2-SNAPSHOT`

File: mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/validation/CompositeValidationStatus.java (hunk 1 above)
Comment (sideisra) [inline mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/validation/CompositeValidationStatus.java]: done

**Pull request merged**